rtmr/extend.go

```diff
@@ -21,22 +21,55 @@ import (
 	"crypto"
 	_ "crypto/sha512" // Register SHA384 and SHA512
 	"fmt"
+	"os"
 
 	"github.com/google/go-configfs-tsm/configfs/configfsi"
 	"github.com/google/go-configfs-tsm/configfs/linuxtsm"
 	"github.com/google/go-configfs-tsm/rtmr"
 )
 
-// ExtendDigestClient extends the measurement to the rtmr with the given client.
+// ExtendDigestSysfs extends the measurement to the RTMR through the sysfs interface.
+// This is the modern method for extending TDX RTMRs.
+func ExtendDigestSysfs(rtmrIndex int, digest []byte) error {
+	// The sysfs file is an interface for RTMR. Each write operation functions
+	// as an "extend" operation, not a normal file write.
```

Comment:
we should check the rtmrIndex is within range (0 ~ 3), as well as the digest length to match a sha384 digest